lib/vanagon/platform/deb.rb

```diff
@@ -21,9 +21,9 @@ def generate_package(project)
       # Method to generate the files required to build a debian package for the project
       #
       # @param workdir [String] working directory to stage the evaluated templates in
-      # @param name [String] name of the project
+      # @param project [Vanagon::Project] project to build a debian package of
       # @param binding [Binding] binding to use in evaluating the packaging templates
-      def generate_packaging_artifacts(workdir, name, binding)
```

Comment:
:) Yeah - I think Mike has voted no on that one.
Must admit that when I saw the list of files that needed to be changed I did baulk a bit.